src/ImmersedBoundaries/grid_fitted_bottom.jl

```diff
@@ -89,21 +89,20 @@ end
 Adapt.adapt_structure(to, ib::GridFittedBottom) = GridFittedBottom(adapt(to, ib.bottom_height), adapt(to, ib.immersed_condition))
 
 """
-    ImmersedBoundaryGrid(grid, ib::GridFittedBottom)
+    numerical_immersed_boundary(grid, ib)
 
-Return a grid with `GridFittedBottom` immersed boundary (`ib`).
-
-Computes `ib.bottom_height` and wraps it in a Field. `ib.bottom_height` is the z-coordinate of top-most interface
-of the last ``immersed`` cell in the column.
+Returns a new `ib` wrapped around a Field that holds the numerical `immersed_boundary`. 
+If `ib` is an `AbstractGridFittedBottom`, `ib.bottom_height` is the z-coordinate of 
+top-most interface of the last ``immersed`` cell in the column. If `ib` is a `GridFittedBoundary`,
+`ib.mask` is a field of booleans that indicates whether a cell is immersed or not.
 """
-function ImmersedBoundaryGrid(grid, ib::GridFittedBottom)
+function numerical_immersed_boundary(grid, ib::GridFittedBottom)
```

Comment:
we use the term "regularize" or "materialize" elsewhere for things like this (ie final processing that occurs ina. constructor)